msf_updates/src/position_msf/init_yaw.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+PACKAGE = 'msf_updates'
+import roslib;roslib.load_manifest(PACKAGE)
+import rospy
+from geometry_msgs.msg import Vector3
+
+import dynamic_reconfigure.client
+
+client = dynamic_reconfigure.client.Client("msf_gps_pose_estimator/position_sensor", timeout=30)
```

Comment:
Move to parameter, and expose via command line args, with the current value as default.